src/store/RelayEnvironment.js

```diff
@@ -28,10 +30,13 @@ const readRelayQueryData = require('readRelayQueryData');
 const relayUnstableBatchedUpdates = require('relayUnstableBatchedUpdates');
 const warning = require('warning');
 
+const noop = function() {};
```

Comment:
We have a helper module for this:

`const emptyFunction = require('emptyFunction');`